charts/aws-fsx-csi-driver/templates/controller-deployment.yaml

```diff
@@ -93,12 +93,14 @@ spec:
           args:
             - --csi-address=$(ADDRESS)
             - --v={{ .Values.sidecars.provisioner.logLevel }}
-            - --timeout=5m
+            - --timeout=$(TIMEOUT)
             - --extra-create-metadata
             - --leader-election=true
           env:
             - name: ADDRESS
               value: /var/lib/csi/sockets/pluginproxy/csi.sock
+            - name: TIMEOUT
```

Comment:
This should be part of a separate commit

Reply:
I made this change in separate commit: https://github.com/kubernetes-sigs/aws-fsx-csi-driver/pull/368/commits/22c8a714ac7fa12b8a5e9f2175a11121b035a826

Sorry for force push 🙇